helpful.el

```diff
@@ -1333,10 +1332,11 @@ LIBRARY-NAME takes the form \"foo.el\" , \"foo.el\" or
 \"src/foo.c\".
 
 If .elc files exist without the corresponding .el, return nil."
-  (when (member (f-ext library-name) '("c" "rs"))
+  (when (member (file-name-extension library-name) '("c" "rs"))
     (setq library-name
-          (f-expand library-name
-                    (f-parent find-function-C-source-directory))))
+          (expand-file-name library-name
+                            (file-name-directory
+                             (directory-file-name find-function-C-source-directory)))))
```

Comment:
This is a bit unfortunate. Emacs 29 has `file-name-parent-directory` but that's rather new.